www/src/pages/docs/actions.js

```diff
@@ -80,7 +96,23 @@ export default ActionCreatorsDocs
 
 export const pageQuery = graphql`
   query {
-    file(relativePath: { eq: "gatsby/src/redux/actions.js" }) {
+    public: file(relativePath: { eq: "gatsby/src/redux/actions/public.js" }) {
```

Comment:
Is split here beneficial? We could maybe do
```suggestion
    public: file(relativePath: { in: ["gatsby/src/redux/actions/public.js", "gatsby/src/redux/actions/restricted.js"] }) {
```
if we just display them all to keep all actions sorted. and adjust `githubPath` separately. We already have second way of providing github path (used in https://www.gatsbyjs.org/docs/node-apis/ / https://github.com/gatsbyjs/gatsby/blob/master/www/src/pages/docs/node-apis.js where almost all APIs are called in different files)

Reply:
yes that makes way more sense!